src/main/java/org/broadinstitute/hellbender/engine/spark/SparkCommandLineArgumentCollection.java

```diff
@@ -45,4 +59,24 @@ public String getSparkMaster() {
         return sparkMaster;
     }
 
+    public String getSparkVerbosity(final Log.LogLevel toolVerbosity) {
+        if (sparkVerbosity != null) return sparkVerbosity;
+        if (toolVerbosity.equals(Log.LogLevel.DEBUG)) {
+            return Level.DEBUG.name();
+        }
+        if (toolVerbosity.equals(Log.LogLevel.INFO)) {
+            return Level.INFO.name();
+        }
+        if (toolVerbosity.equals(Log.LogLevel.WARNING)) {
+            return Level.WARN.name();
+        }
+        if (toolVerbosity.equals(Log.LogLevel.ERROR)) {
+            return Level.ERROR.name();
+        }
+        throw new IllegalStateException("Unknown tool verbosity: " + toolVerbosity.name());
+    }
+
+    public void setSparkVerbosity(String level) {
```

Comment:
Hmm... I don't like having a setter for this argument collection. Looking around it looks like most argument other argument collections aren't tested directly or have public fields. I would say flag this as package protected (remove public) and add the annotation `@VisibleForTesting` to it.

Reply:
Yes needed this for testing and usually I would make the argument variables `public` but I was trying to be consistent with `sparkMaster`. I'll add `@VisibleForTesting`